java/ql/test/kotlin/library-tests/literals/literals.expected

```diff
@@ -15,9 +15,9 @@
 | literals.kt:17:30:17:33 | 15 | LongLiteral |
 | literals.kt:18:30:18:39 | 11 | LongLiteral |
 | literals.kt:19:30:19:38 | 1234567 | LongLiteral |
-| literals.kt:20:32:20:35 | 0.0 | FloatingPointLiteral |
-| literals.kt:21:32:21:37 | 123.4 | FloatingPointLiteral |
-| literals.kt:22:32:22:38 | -123.4 | FloatingPointLiteral |
+| literals.kt:20:32:20:35 | 0.0 | FloatLiteral |
+| literals.kt:21:32:21:37 | 123.4 | FloatLiteral |
+| literals.kt:22:32:22:38 | -123.4 | FloatLiteral |
```

Comment:
Could not verify whether this test still succeeds because test execution from within VS Code failed:
```
extraction error: #redacted#\codeql\java\ql\test\kotlin\library-tests\literals\literals.ql
IOException while executing process with args: [#redacted#\AppData\Roaming\Code\User\globalStorage\github.vscode-codeql\distribution23\codeql\java\tools\kotlin-extractor, --java-home, #redacted#\AppData\Roaming\Code\User\globalStorage\github.vscode-codeql\distribution23\codeql\tools\win64\java, --, -J-Xmx2G, literals.kt]
(eventual cause: IOException "CreateProcess error=193, %1 is not a valid Win32 application")
```

But I assume this is "expected" while Kotlin support is still work in progress.